packages/transport/src/core/TransportController.ts

```diff
@@ -29,7 +29,8 @@ export enum ControllerName {
     RelationshipTemplator = "RelationshipTemplator",
     Secret = "Secret",
     Sync = "Sync",
-    Token = "Token"
+    Token = "Token",
+    PublicRelationshipTemplateController = "PublicRelationshipTemplateController"
```

Comment:
This list is in alphabetical order, please insert correctly after `Message` and before `Relationship`.